### DISCUSSION TITLE
cmp scrollbar not show, help please

### DISCUSSION THREAD

**Discussion by nickp-real** [olimorris/onedarkpro.nvim#191] [Q&A]
![image](https://github.com/olimorris/onedarkpro.nvim/assets/69962410/01431c18-ff6f-4f70-924b-84b10a3921fd)
pmenusbar highlight
![image](https://github.com/olimorris/onedarkpro.nvim/assets/69962410/49f41a54-202d-4690-8f63-0b98dc84d26b)
[cmp config](https://github.com/NickP-real/.dotfile/blob/4b177afd20cb9f5e71e5adff246fda180fd8c147/.config/nvim/lua/plugins/coding.lua#L51-L285)
[onedarkpro config](https://github.com/NickP-real/.dotfile/blob/main/.config/nvim/lua/plugins/colorscheme.lua)

**Comment by olimorris** [comment]
Hi sorry I'm late to this. Did you manage to solve this? Might be worth opening this on the nvim-cmp discussions.

**Reply by nickp-real** [reply]
I will try to open the discussion on nvim-cmp then, thx for the reply btw.
